.travis.yml

```diff
@@ -1,13 +1,13 @@
-sudo: false
 language: go
 go:
-- 1.11.4
-before_install:
-- curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-- if [ ! -d $CODE_DIRECTORY ]; then mkdir -p $HOME/gopath/src/github.com/Nike-Inc; ln -s $TRAVIS_BUILD_DIR $CODE_DIRECTORY; fi # CI for forks
-env:
-  global:
-  - CODE_DIRECTORY=$HOME/gopath/src/github.com/Nike-Inc/cerberus-go-client
+  - 1.15.x
+os:
+  - linux
+dist: xenial
 script:
-- cd $CODE_DIRECTORY # change dir into source
-- make test 2>&1 # Run tests
```

Comment:
I noticed you got rid of the make command. Is the Makefile is this repo still relevant or useful with Go modules? Should it be changeD?

Reply:
good call, I wasn't even thinking about the make file. I don't think we need it any more since we can just say "go test <sub module>"